src/Turbo/README.md

```diff
@@ -0,0 +1,487 @@
+# Symfony UX Turbo
+
+Symfony UX Turbo is a Symfony bundle integrating the [Hotwire Turbo](https://turbo.hotwire.dev)
+library in Symfony applications. It is part of [the Symfony UX initiative](https://symfony.com/ux).
+
+Symfony UX Turbo allow having the same user experience as with [Single Page Apps](https://en.wikipedia.org/wiki/Single-page_application)
+but without having to write a single line of JavaScript!
+
+Symfony UX Turbo also integrates with [Symfony Mercure](https://symfony.com/doc/current/mercure.html) to broadcast DOM changes
+to all currently connected users!
+
+You're in a hurry? Take a look to [the chat example](#sending-async-changes-using-mercure-a-chat) to discover the full potential
+of Symfony UX Turbo.
+
+## Installation
+
+Symfony UX Turbo requires PHP 8+ and Symfony 4.4+.
+
+Install this bundle using Composer and Symfony Flex:
+
+```sh
+composer require symfony/ux-turbo
+
+# Don't forget to install the JavaScript dependencies as well and compile
+yarn install --force
```

Comment:
why force?

Reply:
It's like that in all readme fils of this repo.